MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include AUTHORS
 include ChangeLog
 include LICENSE
+graft tests
```

Comment:
Looking at other repos and what PBR produces by doing a graft here if the tests have been run locally (which is likely) graft will pick up the `*.pyc`/`__pycache__` files and include them into the sdist. requests and others use: `recursive-include tests *.py` to get around this. 

In our case [PBR](https://docs.openstack.org/pbr/latest/) will take care of this because it knows where our tests folder is located it'll include just the `*.py` folder into the sdist. With pyc files in the folder an nothing in MANIFEST.in I get what i expect in the sdist.  

Please drop this change.